add confluence link to how-to-contribute page (#382)
apache/accumulo-website: pages/how-to-contribute.md

there are other ways to contribute to Accumulo:

* communicate on one of the [mailing lists](/contact-us/#mailing-lists)

> * participate in design discussions on [Confluence][confluence]

Reviewer: I would not characterize Confluence as a place for "discussions". The mailing list is for that. Confluence is for drafting/posting/commenting-on design docs.

I suggest:

```suggestion
* view/draft/comment on designs or other docs in [Confluence][confluence]
```

## Accumulo Repositories

| Repository         | Links                          | Description |
| -------------------| ------------------------------ | ----------- |
| [Accumulo][a]      | [Contribute][ac] [Issues][ai]  | Core Project |
| [Website][w]       | [Contribute][wc] [Issues][wi]  | Source for this website |
| [Examples][e]      | [Contribute][ec] [Issues][ei]  | Example code |
| [Testing][t]       | [Contribute][tc] [Issues][ti]  | Test suites such as continuous ingest and random walk |
| [Docker][d]        | [Contribute][dc] [Issues][di]  | Source for Accumulo Docker image |
| [Wikisearch][s]    | [Contribute][sc] [Issues][si]  | Example application that indexes and queries Wikipedia data |
| [Proxy][p]         | [Issues][pi]                   | Apache Thrift service that exposes Accumulo to other languages |

> | [Maven plugin][m]  | [Issues][mi]                   | Maven plugin that runs Accumulo |

Reviewer: Since these are optional, I would take it or leave it. I don't think it matters, as long as it's consistent. I think I recently fixed a few lines on the people page that were missing these, but only because all the other lines had them, not because they are needed.

I'm okay if you leave this change in, but it's also fine to leave it unchanged. I don't think it matters.

Author: This resolves a warning about possible incompatibility between versions - so it seemed safer to include them, even if optional.

Reviewer: Sounds good. What warning, though? Was this in the Jekyll build output, or in a Markdown-aware text editor you use?

Author: IDE.  

```
For compatibility reasons all table rows should have borders (pipe symbols) at the start and at the end
```

Reviewer: For what it's worth, we specifically use Kramdown for GitHub-flavored-markdown (gfm), which is what Jekyll uses. So we don't really require compatibility with restrictive variants of Markdown that the IDE is trying to warn about. But, if it gets rid of the IDE error, I'm fine with it. I don't ever use an IDE for Markdown (or for the website at all). I'm a little surprised anybody does. I suspect it might be a bit challenging to get it working well in an IDE, given the mix of Ruby, Bundler, Jekyll, HTML, CSS, JavaScript, Markdown, Liquid, YAML, and who knows how many other languages are mixed in that I missed. :smiley_cat:

Author: Well, for the spell check alone I find it worth it.  Does not help when I end up with a correctly spelled, but incorrect word but maybe chatGPT will come to my rescue someday.

IntelliJ does provide a preview pain that does give a good rough idea of what the render looks like - some things for our pages don't work, but what does helps getting a rough idea of how things are rendered, tables, font sizes and highlighting which I do find useful.

* **Build resources** - [Jenkins][jenkins]
* **Releases** - [Making a release][making], [Verifying a release][verifying]

> * **Confluence** - [Accumulo Confluence Space][confluence] provides a collaboration space for design documents.  If you would like write
> access, please send a request to `dev@accumulo.apache.org` with your Confluence username.

Reviewer: I would characterize this as a "require" rather than a "want", because there should be an actual need. Characterizing it as a need, implies that the requester provide some answer to "why?" up front, rather than us asking it as a follow-up question. So, instead of "I want write access" (and us having to ask "for what?"), it becomes "I require write access because I want to contribute to document X".

```suggestion
* **Confluence** - [Accumulo Confluence Space][confluence] provides a collaboration space for design documents. If you require write
access, please [contact us][contact] with your Confluence username and request details.
```
